nva-datamodel-java/src/main/java/no/unit/nva/model/associatedartifacts/file/File.java

```diff
@@ -77,8 +81,9 @@ public abstract class File implements JsonSerializable, AssociatedArtifact {
     private final URI license;
     @JsonProperty(ADMINISTRATIVE_AGREEMENT_FIELD)
     private final boolean administrativeAgreement;
-    @JsonProperty(PUBLISHER_AUTHORITY_FIELD)
-    private final boolean publisherAuthority;
+
+    @JsonProperty(PUBLISHER_VERSION)
```

Comment:
Can you call it PUBLISHER_VERSION_FIELD to avoid the conflict of naming with the object and the field name+